samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java

```diff
@@ -51,11 +57,12 @@
 
         // Respond to the user.
         return turnContext
-            .sendActivities(MessageFactory.text(responseText), createSuggestedActions())
-            .thenApply(responses -> null);
+            .sendActivity(MessageFactory.text(responseText))
+            .thenCompose(responses -> sendSuggestedActions(turnContext))
```

Comment:
```suggestion
            .thenCompose(response -> sendSuggestedActions(turnContext))
```

As it's `sendActivity`